src/Command/PullRequest/PullRequestSwitchBaseCommand.php

```diff
@@ -34,9 +34,8 @@ protected function configure()
             ->addArgument('pr_number', InputArgument::REQUIRED, 'PR number to be switched')
             ->addArgument(
                 'base_branch',
-                InputArgument::OPTIONAL,
-                'Name of the new base branch to switch the PR to',
-                'master'
+                InputArgument::REQUIRED,
+                'Name of the new base branch to switch the PR to'
```

Comment:
The base_branch argument is now required, but this seemed to be a bug as you don't want to auto switch to master always. I a follow-up pr we may consider making this optional and showing a selection menu when missing 👍